pkg/arvo/app/link-store.hoon

```diff
@@ -0,0 +1,172 @@
+::  link: social bookmarking
+::
+::    the paths under which links are submitted are generally expected to
+::    correspond to existing group paths. for strictly-local collections of
+::    links, arbitrary paths are probably fair game, but could trip up
+::    primitive ui implementations.
+::
+::  scry and subscription paths:
+::
+::    /local-pages/[some-group]    all pages we saved by recency
+::    /submissions/[some-group]    all submissions by recency
+::
+/+  *link, default-agent, verb
+::
+|%
++$  state-0
```

Comment:
why name it 0 instead of zero? curious, since it seems uncommon for us to include numbers in variable names

Reply:
imo using words in place of digits for versioning is terrible. `state-0` is in line with its tag, `%0`, and also happens to keep width-consistency with `-1` through `-9`.

Reply:
Personally I generally use words for numbers when there's only a particular set of them (where it's really an enum) and numerals where they're actually numbers (like here where they can go on forever).